config.yaml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
(not related to this file)
We'll want to add a step to the CI to publish the charm to charmhub. Could be in a different PR.

Reply:
We may also want to add Codeql analysis

Reply:
Cool